website/content/landscape/SustainabilityUseCasesAndLandscape2023.ko.md

```diff
@@ -33,7 +33,8 @@ toc_hide: true
 
 이 문서에 기여해주신 분들께 특별히 감사드립니다. 콘텐츠를 개선하고 강화하는 데 관심이 있으신 분은 저장소에 PR을 제출하고 아래에서 자신을 기여자로 추가해 주세요!
 
-Huamin Chen, [Marlow Weston](https://github.com/catblade), [Niki Manoledaki](https://github.com/nikimanoledaki), Eun Kyung Lee, [Chen Wang](https://github.com/wangchen615), [Chris Lloyd-Jones](https://github.com/Sealjay), [Parul Singh](https://github.com/husky-parul), [Przemysław Perycz](https://github.com/pperycz), [Christopher Cantalupo](https://github.com/cmcantalupo), [Patricia Cahill](https://github.com/patricia-cahill), [Jochen Joswig](https://github.com/by-d-sign), [Emily Fox](https://github.com/thefoxatwork), [Leonard Pahlke](https://github.com/leonardpahlke)\
+<!-- cspell:disable-next-line -->
```

Comment:
how cspell integration with i18n?

Reply:
cspell does spell checking towards currently supported dictionaries, where only English (Great Britain and US) language dictionary is supported as of now. It will ignore the rest, like Korean for example.

Reply:
got it.